eds4jinja2/builders/jinja_builder.py

```diff
@@ -7,19 +7,29 @@
 
 import jinja2
 
+from eds4jinja2.adapters.tabular_utils import invert_dict, replace_strings_in_tabular, add_relative_figures
 from eds4jinja2.adapters.file_ds import FileDataSource
 from eds4jinja2.adapters.sparql_ds import SPARQLEndpointDataSource
 
-FROM_ENDPOINT = "from_endpoint"
-FROM_FILE = "from_file"
-
 DATA_SOURCE_BUILDERS = {
-    FROM_ENDPOINT: lambda endpoint: SPARQLEndpointDataSource(endpoint),
-    FROM_FILE: lambda file_path: FileDataSource(file_path)
+    "from_endpoint": lambda endpoint: SPARQLEndpointDataSource(endpoint),
```

Comment:
why did you decide to remove the constants?

Reply:
just realised they are superfluous. this is more readable and practical.